tcf_website/static/course/course.css

```diff
@@ -0,0 +1,5 @@
+#course-eval-link{
```

Comment:
Is it better to go by class name instead of id? I noticed for most css files we go by class name.

Reply:
class name is best used for styles that you need to apply to multiple similar elements!

For a one-off thing we usually recommend bootstrap classes instead of custom css. Do you think you can modify this?

Reply:
class name is best used for styles that you need to apply to multiple similar elements!

For a one-off thing we usually recommend bootstrap classes instead of custom css. Do you think you can modify this?